### PR TITLE
Do not use deprecated app-id

### PR DESCRIPTION
**Pull request by sonnyp**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-.flatpak-builder
\ No newline at end of file
+.flatpak-builder
+.flatpak
```
Comment (sonnyp) [inline .gitignore]: `.flatpak` is created by VSCode Flatpak extension

File: flatpak/design.chris_wood.IconBear.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "app-id" : "design.chris_wood.IconBear",
+    "id" : "design.chris_wood.IconBear",
```
Comment (sonnyp) [inline flatpak/design.chris_wood.IconBear.json:2]: `app-id` is deprecated

**Comment by azuredusk10** [comment]
Thanks for this Sonny! I also noticed this issue. I moved the "Import" button into the header bar itself in the branch where I am refactoring the import dialog. I don't think that particular fix will be needed anymore once I merge the branch into master, but the other 2 changes look great - thank you

**Comment by sonnyp** [comment]
Removed the change in `window.js`

**Pull request merged**